src/main/java/online/talkandtravel/model/dto/chat/NewSubChatRequest.java

```diff
@@ -0,0 +1,5 @@
+package online.talkandtravel.model.dto.chat;
+
+public record NewSubChatRequest(String name, String description, String countryId) {
```

Comment:
this dto is used for POST request, so it should have some field constraints to validate. Add some annotations with some constraints, and turn on validation process by adding @Valid in controller